packages/core/app/components/utils/useValues.js

```diff
@@ -135,6 +135,18 @@ function useLocalStorageState(key, initialState, clean) {
   return [value, setValue, remove];
 }
 
+function clearLocalStorage() {
```

Comment:
Should this also take an optional argument so it can be used to delete a very specific local storage entry?

Reply:
✅